src/implementations/twig/components/file/file.story.js

```diff
@@ -210,10 +210,11 @@ const prepareData = (data, args) => {
     data.description = '';
   }
 
-  if (args.show_image && data.image) {
-    data.image.src = args.image;
-  } else if (!args.show_image && data.image) {
-    data.image = {};
+  if (args.show_image && data.picture) {
+    data.picture.img = {};
```

Comment:
you are right, i fixed this but at the same time i find it crazy the way we handle this data in order to offer controls for everything, it adds complexity where it would not really be needed, we should step out from this approach, in my opinion, for ECL v4